go.mod

```diff
@@ -198,7 +198,9 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.15.0-sdk-v0.46.13
+	// SDK with blocktime in GRPC query: https://github.com/celestiaorg/cosmos-sdk/pull/337
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.15.0-sdk-v0.46.13.0.20230731181258-40cbd617f1ba
```

Comment:
[blocking] thanks for the fix in celestiaorg/cosmos-sdk! If possible, I think we should hold off on merging this PR until we get that PR merged and cut an official release of celestiaorg/cosmos-sdk